app/pages/index.js

```diff
@@ -13,10 +14,20 @@ export default withData(props => (
         body {
           margin: 2em;
         }
+        section {
+          height: calc(50vh - 4em);
+        }
       `}</style>
     </Head>
-    <h1>Apollo upload example</h1>
-    <p>Select an image to upload and view the response in the console.</p>
-    <SingleUploader />
+    <section>
+      <h1>Apollo upload example (single file)</h1>
```

Comment:
Lets have a h1 `Apollo upload examples` above the sections, and simplify the text of the section headings and make them h2.